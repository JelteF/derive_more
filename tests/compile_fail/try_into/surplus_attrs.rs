#[derive(derive_more::TryInto)]
#[try_into(owned)]
#[try_into(foo)]
enum MixedData {
    Int(u32),
    String(String),
}

fn main() {}
