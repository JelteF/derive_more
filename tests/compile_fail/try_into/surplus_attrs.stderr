error: Only a single attribute is allowed.
       For the top-level attribute, an additional `#[try_into(error(...))]` may be provided.
 --> tests/compile_fail/try_into/surplus_attrs.rs:3:1
  |
3 | #[try_into(foo)]
  | ^
